front/lib/models/assistant/actions/mcp.ts

```diff
@@ -75,6 +79,10 @@ AgentMCPServerConfiguration.init(
         key: "id",
       },
     },
+    internalMCPServerId: {
```

Comment:
These ids have could change a bit over time? This will mean migrating all agent configuration? Can you give a bit more substance on the added value to better understand the trade-off?

Reply:
Yes